Numberdrome.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+class Numberdrome {
+  constructor() {
+    this.numbers = [];
+  };
```

Comment:
add empty lines between class methods
do not need to add `;` after calss method `}`